src/documents/views.py

```diff
@@ -362,7 +362,9 @@ def thumb(self, request, pk=None):
                 handle = doc.thumbnail_file
             # TODO: Send ETag information and use that to send new thumbnails
             #  if available
-            return HttpResponse(handle, content_type="image/png")
+            content_type = "image/webp"
+
+            return HttpResponse(handle, content_type=content_type)
```

Comment:
perhaps just inline it as the `content_type` variable is not used anywhere else, so:

`return HttpResponse(handle, content_type="image/webp")`

Reply:
Right, one last bit I missed there.